lib/member_page.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class MemberPage < Scraped::HTML
+  field :birth_date do
+    date_from(
+      record_for('Datël')
+    ).to_s
+  end
+
+  field :birth_place do
+    record_for('Vendlindja:')
+  end
+
+  field :area do
+    record_for('Qarku:')
+  end
+
+  # TODO: This isn't so easy to extract as the text varies
+  # field :commissions do
+  #   puts line_for('Komisioni').split(/(Komisioni)/).drop(1).join('').tidy
+  # end
+
+  private
+
+  def record_for(str)
+    table.xpath(".//span[contains(., '#{str}')]")
+         .text
+         .split(':')
+         .drop(1)
+         .last
```

Comment:
This is to handle cases where the record heading is present (for example 'Vendlindja:') but isn't followed by the actual record. This is being tested in: 13fc0eb0cfeb6e1d1647b3f1f3e25191354c4817

I've replaced `split(':').drop(1).last` with your example.